app/javascript/packs/components/VolunteerMap.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import GoogleMapReact from "google-map-react";
+import Box from "@material-ui/core/Box";
+import { makeStyles } from "@material-ui/core/styles";
+import MapMarker from "./icons/MapMarker";
+import useWindowSize from "../hooks/useWindowSize";
+import mapStyles from "../helpers/mapStyles";
+import volunteerLocations from "../helpers/volunteerLocations";
+
+const useStyles = makeStyles((theme) => ({
+  mapWrapper: {
+    paddingBottom: "70%",
+    "@media (min-width:980px)": {
+      paddingBottom: "50%",
+    },
+    "@media (min-width:1280px)": {
+      paddingBottom: "35%",
+    },
+    position: "relative",
+    "& > *": {
+      position: "absolute !important",
+      top: "0",
+      left: "0",
+      width: "100%",
+      height: "100%",
+    },
+  },
+}));
+
+const mapOptions = {
+  styles: mapStyles,
+};
+
+const getZoomByWindowWidth = (windowWidth) => {
+  switch (true) {
+    case windowWidth > 1200: {
+      return 4.6;
+    }
+    case windowWidth > 700: {
+      return 4.2;
+    }
+    default: {
+      return 3.3;
+    }
+  }
+};
+
+const VolunteerMap = ({ center }) => {
+  const classes = useStyles();
+  const [windowWidth] = useWindowSize();
+  return (
+    <Box width="100%" className={classes.mapWrapper}>
+      <GoogleMapReact
+        bootstrapURLKeys={{ key: "AIzaSyC91p5MR0B-sProwqXqStywWfAXOxOkcm8" }}
```

Comment:
(Reminder to add our prod and staging domains to this key besides localhost.)

Reply:
> > Looks awesome! Should we jiggle the lats and lngs so we don't release detailed location info?
> 
> That's a great idea! I'm actually not using our volunteer data, I snagged some fake locations from https://www.latlong.net/category/cities-236-15.html
> 
> I think the volunteer Airtable form only collects City/State/Region so I don't think we have to worry about detailed location info.

@cantino Keith gave me an export of the vounteers. They are indeed only cities.